sys/disp_dev/Makefile

```diff
@@ -0,0 +1 @@
+include $(RIOTBASE)/Makefile.base
```

Comment:
why `sys_...`? it's the first module that does this. Maybe better rename to just `disp_dev` and optionally propose proper sorting?

Reply:
I can move the module to `drivers` if you prefer. That would make more sense actually.

Reply:
Ah, I just meant the name of the test, commented on the wrong file. I mean, `tests/sys_disp_dev` vs `tests/disp_dev`.

Reply:
Renamed